frontend/src/constants/constants.ts

```diff
@@ -14,11 +14,13 @@ export const MTL_MAX_LATITUDE = 45.752154;
 export const MAPBOX_GEOCODING_API =
   "https://api.mapbox.com/geocoding/v5/mapbox.places/";
 export const MAPBOX_STYLE_URL = "mapbox://styles/mumap/ckhb8l6je01ki1ao4tabh8e2q";
-export const GET_ALL_ARTISTS_API = "http://localhost:3000/artist";
-export const GET_ALL_BOROUGH_API = "http://localhost:3000/borough";
-export const CREATE_MURAL_API = "http://localhost:3000/mural";
+export const GET_ALL_ARTISTS_API = `${process.env.REACT_APP_BACKEND_URL}/artist`;
+export const GET_ALL_BOROUGH_API = `${process.env.REACT_APP_BACKEND_URL}/borough`;
+export const CREATE_MURAL_API = `${process.env.REACT_APP_BACKEND_URL}/mural`;
+export const GET_ALL_COLLECTION = `${process.env.REACT_APP_BACKENDURL}/collection`;
 export enum FORM {
   MURAL = "Mural",
   COLLECTION = "Collection",
   TOUR = "Tour"
 };
+export const BACKENDURL = process.env.REACT_APP_BACKEND_URL;
```

Comment:
we should be able to remove this if it's unused

Reply:
removed